rwiki-tool/tool/src/bundle/uk/ac/cam/caret/sakai/rwiki/tool/bundle/Messages_en_GB.properties

```diff
@@ -179,6 +179,8 @@ jsp_incomming_pages_help = This is a list of pages that reference or link to thi
 
 jsp_info = Info
 
+jsp_info_breadcrumb = Information for
```

Comment:
If this is a new property it should go in the standard one. Otherwise if you're not running in en_GB you won't see a value.